server/events/vcs/azuredevops_client.go

```diff
@@ -124,43 +125,82 @@ func (g *AzureDevopsClient) CreateComment(repo models.Repo, pullNum int, comment
 	return nil
 }
 
-// PullIsApproved returns true if the merge request was approved.
-// https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops#require-a-minimum-number-of-reviewers
+// PullIsApproved returns true if the merge request was approved by another reviewer.
 func (g *AzureDevopsClient) PullIsApproved(repo models.Repo, pull models.PullRequest) (bool, error) {
+	owner, project, repoName := SplitAzureDevopsRepoFullName(repo.FullName)
+
 	opts := azuredevops.PullRequestGetOptions{
 		IncludeWorkItemRefs: true,
 	}
-	owner, project, repoName := SplitAzureDevopsRepoFullName(repo.FullName)
 	adPull, _, err := g.Client.PullRequests.GetWithRepo(g.ctx, owner, project, repoName, pull.Num, &opts)
 	if err != nil {
-		return false, errors.Wrap(err, "getting pull request")
+		return false, fmt.Errorf("get pull request: %w", err)
 	}
+
 	for _, review := range adPull.Reviewers {
 		if review == nil {
 			continue
 		}
-		if review.GetVote() == azuredevops.VoteApproved {
+
+		if review.IdentityRef.GetUniqueName() == adPull.GetCreatedBy().GetUniqueName() {
```

Comment:
Only consider it approved if there exists an approver _other_ than the author

Reply:
I think we should still allow self-approval since it is a valid branch policy.  This causes a confusing error message to a user if they're expecting apply to run successfully.  Could we change this to verify whether this case is supposed to pass the branch policy?

Reply:
Otherwise, looking good!  Thanks for working on this.

Reply:
I mostly made the change to follow along with how Atlantis describes how the `approved` requirement works (https://www.runatlantis.io/docs/apply-requirements.html#supported-requirements)

Thinking aloud.. would it make more sense in this case to only have `mergeable` set as an apply requirement? If your repository has a minimum reviewer count policy, `mergeable` would catch it. If you don't have any policies, `approved` would require two reviewers.

There does seem to be some blurring between the lines here with AzDO..

Reply:
Yes, that approach makes sense to me and it looks like that will make it match the current behavior and documentation for bitbucketcloud:

https://www.runatlantis.io/docs/apply-requirements.html#supported-requirements

Reply:
Also, I ran a few tests in our tenant and all looks good to me.